client/stylesheets/_toCat.scss

```diff
@@ -2025,18 +2045,35 @@ i:hover {
     margin: 3rem 0 1rem 0;
 }
 
+.helpCenterThree {
+    border-top: 1px solid #404040;
+    text-align: left !important;
+}
+
+.helpThree > li {
+    width: 28%;
+    display: inline-block;
+}
+
+.helpThree li ul li h3 {
+    margin: 5px 0 4px 0;
+}
+
+.helpThree li ul li p {
+    margin: 0 0 2px 0;
+}
 /**********
  Contact
  **********/
 
-//  .contactdiv {
-//     text-align: center;
-// }
+.contactdiv {
+    text-align: center;
+}
 
-.contactname input, .contactemail input {
- width: 30%;
- height: 20px;
- padding-left: 5px; 
+.contactname input, .contactsubject input .contactemail input {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Each selector in a comma sequence should be on its own line